contrib/pkg/installmanager/installmanager.go

```diff
@@ -620,29 +620,24 @@ func uploadAdminPassword(cd *hivev1.ClusterDeployment, m *InstallManager) (*core
 	return s, uploadSecretWithRetries(s, m)
 }
 
-func uploadConfigmapWithRetries(c *corev1.ConfigMap, m *InstallManager) error {
-	uploaded := false
-	var err error
+func uploadConfigMapWithRetries(c *corev1.ConfigMap, m *InstallManager) error {
+	backoff := retry.DefaultBackoff
+	backoff.Steps = 10
+	backoff.Duration = time.Second
 
-	// try 10 times
-	for i := 1; i < 11; i++ {
-		err = m.DynamicClient.Delete(context.Background(), c)
-		if err != nil && !errors.IsNotFound(err) {
-			m.log.WithError(err).WithField("configmap", c.Name).Warningf("error deleting existing configmap")
+	err := wait.ExponentialBackoff(backoff, func() (bool, error) {
+		if err := m.DynamicClient.Delete(context.Background(), c); err != nil && !errors.IsNotFound(err) {
+			m.log.WithError(err).WithField("configmap", c.Name).Warning("error deleting existing configmap")
```

Comment:
@joelddiaz shouldn't you return at this point if the error is something other than not found?

Reply:
i suppose that Create() below doesn't stand much of a chance :) updated PR